src/zendoo/mcTestCall.cpp

```diff
@@ -18,8 +18,6 @@
  *  "custom_fields_list_len", "custom_field_0", ... , "custom_field_1"
  *  Generates a TestCertificateProof;
  *
- *  NOTE: "constant" param must be present if "cert" has been passed; If "cert_no_const" has been passed,
```

Comment:
No lol this comment was ok. In line 16 there's still "constant_presence" param. That one must be removed